### DISCUSSION TITLE
新增功能建议

### DISCUSSION THREAD

**Discussion by Aronaten** [FloatTech/ZeroBot-Plugin#940] [Ideas]
功能名：生日祝贺
功能具体：增加一个可以查询最近几天二次元人物生日，并在那一天发送生日祝贺（发送生日祝贺可能有些群会烦，所以希望增加个允许发送生日祝贺和不允许发送生日祝贺）我语文不好，不知道各位开发者大大听懂了没有？

**Comment by xyy0411** [comment]
> 功能名：生日祝贺 功能具体：增加一个可以查询最近几天二次元人物生日，并在那一天发送生日祝贺（发送生日祝贺可能有些群会烦，所以希望增加个允许发送生日祝贺和不允许发送生日祝贺）我语文不好，不知道各位开发者大大听懂了没有？

二次元角色那么多，那岂不是每天都要播报😧
